Component/Adapter/DoctrineCacheAdapter.php

```diff
@@ -33,7 +34,14 @@ public function __construct(CacheProvider $provider)
      */
     public function getItem($key)
     {
-        return $this->provider->fetch($key);
+        if (true === $this->provider->contains($key)) {
```

Comment:
Do to the consistency of coding style within this component, I'm gonna leave this "truthful" Yoda condition and (maybe) refactor it throughout the component later.